README.md

```diff
@@ -59,34 +60,29 @@ class Shot
   attribute :id, Integer
   attribute :name, String
   attribute :url, String
-  attribute :image_url, String
+  attribute :image_url, String, as: :image # You can define custom attribute mapping
```

Comment:
To keep the same line of thought, `CustomAttributeMapping` could be named `AttributeAliasing`.